docs/start/install.mdx

```diff
@@ -8,6 +8,14 @@ description: Install and set up Qiskit and Qiskit Runtime on various operating s
 
 Whether you will work locally or in a cloud environment, the first step for all users is to install Qiskit. For those wanting to run on a real system, your next step is to choose one of two channels in order to access IBM Quantum systems: IBM Quantum Platform or IBM Cloud.
 
+<span id="qiskit-upgrade"></span>
+## Upgrade to Qiskit 1.0
+
+<Admonition type="info" title="You must create a new virtual environment to install Qiskit 1.0">
+  Because Qiskit 1.0 uses a new packaging structure, you cannot upgrade an existing Qiskit 0.x installation to Qiskit 1.0 in-place. Future updates starting with Qiskit 1.0 will allow for in-place upgrades.  
```

Comment:
```suggestion
  Because Qiskit 1.0 uses a new packaging structure, you cannot upgrade an existing Qiskit 0.x installation to Qiskit 1.0 in-place. That is, you must fully delete your virtual environment and completely reinstall it when upgrading.
  
  Once you've upgraded to Qiskit 1.0, you'll be able to upgrade in-place like normal again.
```

Reply:
Thanks @Eric-Arellano  -- I'm still in the draft stage (as marked) and am just copying and pasting from the blog as a placeholder -- would you mind waiting till it's in review to give comments? :D

Reply:
Ah sure, my bad!

Reply:
No problem! I appreciate your close attention to detail. I'll tag you as reviewer when I'm ready 😃